crates/iota-core/src/congestion_tracker.rs

```diff
@@ -18,9 +18,14 @@ use crate::execution_cache::TransactionCacheRead;
 /// Capacity of the congestion tracker's cache.
 const CONGESTION_TRACKER_CACHE_CAPACITY: u64 = 10_000;
 
-/// Alias type for holding transaction's gas price and mutable (or
-/// congested) shared objects.
-type TransactionGasPriceMutSharedObjectsPair = (u64, Vec<ObjectID>);
+/// Threshold for hotness below which an object is considered cold.
+const HOTNESS_CUTOFF: f64 = 1.0;
+
+/// Controls how quickly congestion feedback updates object hotness.
+const HOTNESS_ADJUSTMENT_FACTOR: f64 = 0.2; // > 0.0
```

Comment:
Did you mean congestion tracker? Also, I would suggest putting accepted values directly in the docstring.

```suggestion
/// Controls how quickly congestion tracker updates object hotness.
/// Values should be > 0.0.
const HOTNESS_ADJUSTMENT_FACTOR: f64 = 0.2;
```